packages/shared/lib/core/wallet/classes/activity.class.ts

```diff
@@ -91,9 +91,7 @@ export class Activity implements IActivity {
 
         this.storageDeposit = Number(output.amount) - Number(outputOptions.amount)
         this.expirationDate = new Date(Number(outputOptions?.unlocks?.expiration?.unixTime) * MILLISECONDS_PER_SECOND)
-        this.isAsync =
-            this.storageDeposit > 0 ||
-            !!(outputOptions?.unlocks?.expiration?.milestoneIndex || outputOptions?.unlocks?.expiration?.unixTime)
+        this.isAsync = this.storageDeposit > 0 || !!outputOptions?.unlocks?.expiration?.unixTime
```

Comment:
```suggestion
        this.isAsync = this.storageDeposit > 0 || !!outputOptions?.unlocks?.expirationUnixTime
```

Reply:
There is a type error thrown this way, but it is fixed in wallet-stardust-alpha-2..0.1. It's already released, so you can try it out if you want.